src/ecalc/libraries/libecalc/common/libecalc/input/validation/json_schemas/ecalc-model.json

```diff
@@ -1,40 +0,0 @@
-{
```

Comment:
We may want to have the static json files (as long as they are aligned with the python code; can e.g. add a test on this?) for simpler use?

Reply:
We can generate json files, if that is what you mean. If you are talking about a test while converting I did that, and removed it when I saw it worked similarly.